Typo "**@***Example.Microsoft***.com**"→"**\@***Example.Microsoft***.…

hyoshioka0128 opened the pull request:
…com**"

bold with escape characters

https://docs.microsoft.com/en-us/windows/win32/secauthn/user-name-formats

hyoshioka0128 committed:
Typo "**@***Example.Microsoft***.com**"→"**\@***Example.Microsoft***.com**"

bold with escape characters

https://docs.microsoft.com/en-us/windows/win32/secauthn/user-name-formats

PRMerger8 commented:
@hyoshioka0128 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

hkhmsft committed:
Update user-name-formats.md

Fixed and removed nested asterisks and replaced with tags to preserve intended formatting.

PRMerger12 commented:
@lastnameholiu : Thanks for your contribution! The author(s) have been notified to review your proposed change.

hkhmsft commented:
Thanks for your contribution, I've merged your changes and they should be live soon.
